Example/MediaView.xcodeproj/project.pbxproj

```diff
@@ -505,6 +505,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
 				MTL_ENABLE_DEBUG_INFO = NO;
+				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
```

Comment:
Needed for Travis. Scheme should set this to `NO` when building framework for release.